internal/sql/parser/parser.go

```diff
@@ -268,6 +268,9 @@ func (p *Parser) parseTokens(tokens ...scanner.Token) error {
 // present, it unscans and return false. If the first is present, all the others
 // must be parsed otherwise an error is returned.
 func (p *Parser) parseOptional(tokens ...scanner.Token) (bool, error) {
+	if len(tokens) == 0 {
```

Comment:
This fixes a potential panic if tokens are empty.

Reply:
Panics are good in these cases, we don't want to use this method with empty arguments